src/WebRTCSession.cpp

```diff
@@ -382,51 +606,134 @@ addDecodeBin(GstElement *webrtc G_GNUC_UNUSED, GstPad *newpad, GstElement *pipe)
         gst_object_unref(sinkpad);
 }
 
-std::string::const_iterator
-findName(const std::string &sdp, const std::string &name)
+bool
+strstr_(std::string_view str1, std::string_view str2)
```

Comment:
```suggestion
static bool
contains(std::string_view str1, std::string_view str2)
```

Reply:
I can change the function name. static isn't required as it's inside an anonymous namespace.

Reply:
Woops, didn't see that one. I think changing the name still makes it clearer, since my C days are long gone, thanks! :D